egret/models/dcopf.py

```diff
@@ -229,6 +255,26 @@ def create_ptdf_dcopf_model(model_data, include_feasibility_slack=False, base_po
         p_marginal_slack_penalty = _validate_and_extract_slack_penalty(md)                
         p_rhs_kwargs, penalty_expr = _include_system_feasibility_slack(model, gen_attrs, bus_p_loads, p_marginal_slack_penalty)
 
+    if dc_branches:
+        dcpf_bounds = dict()
+        dc_loss_multiplier = dict()
```

Comment:
I don't see where this is used within the "if dc_branches" conditional.

Reply:
It's not -- was part of a slightly more complex model which is not part of this PR.